cellprofiler/modules/threshold.py

```diff
@@ -498,9 +546,9 @@ def create_settings(self):
         self.adaptive_window_size = cellprofiler.setting.Integer(
             "Size of adaptive window",
             50,
-            doc="""
-            Enter the window for the adaptive method. For example, you may want to use a multiple of the
-            largest expected object size.
+            doc="""\
+Enter the window for the adaptive method. For example, you may want to use a multiple of the
```

Comment:
Please add `*(Used only if "{TS_ADAPTIVE}" is selected for thresholding scope)*` and a newline.